resources/views/admin/admin_my_profile.blade.php

```diff
@@ -0,0 +1 @@
+fghjkl
```

Comment:
What is this? and why is this here?

Reply:
All files were deleted because they were unnecessary.
I will try to confirm the deletion in the future.